src/components/editTestCase/groupCoverage/GroupCoverage.tsx

```diff
@@ -0,0 +1,225 @@
+    );
+    if (groupCalculations) {
+      const relevantPopulations = groupCalculations.populationRelevance;
+      const statementResults = groupCalculations.statementResults;
+      return relevantPopulations.map((population) => {
+        const populationResult = statementResults.find(
+          (statementResult) =>
+            statementResult.statementName === population.criteriaExpression
+        );
+        return {
+          ...populationResult,
+          populationName: FHIR_POPULATION_CODES[population.populationType],
+        };
+      });
+    }
+    return [];
+  };
+
+  const changeCriteria = (criteriaId: string) => {
+    setSelectedCriteria(criteriaId);
+    const populationResults = getPopulationResults(criteriaId);
+    setPopulationResults(populationResults);
+    const group = groupPopulations.find((gp) => gp.groupId === criteriaId);
+    setSelectedPopulation(getFirstPopulation(group));
+  };
+
+  const changePopulation = (population: Population) => {
+    setSelectedPopulation(population);
+    const result = populationResults?.find(
+      // TODO: Handle 2 IP scenario
+      (result) => result.populationName === population.name
+    );
+    setCoverageHtml(
+      result ? result.statementLevelHTML : "No results available"
+    );
+  };
+
+  const getCriteriaLabel = (index) => {
+    return groupPopulations.length > 1
+      ? `${populationCriteriaLabel} ${index + 1}`
+      : populationCriteriaLabel;
+  };
+
+  const populationCriteriaOptions = [
+    ...groupPopulations?.map((gp, index) => {
+      return (
+        <MenuItem
+          key={gp.groupId}
+          value={gp.groupId}
+          data-testid={`option-${gp.groupId}`}
+        >
+          {getCriteriaLabel(index)}
+        </MenuItem>
+      );
+    }),
+  ];
+  return (
+    <>
+      <div tw="border-b pb-2">
+        <Select
+          id={"population-criterion-selector"}
+          tw="w-1/4"
+          sx={{
+            height: "32px",
+            borderColor: "transparent",
+            "& .Mui-focused": {
+              borderColor: "transparent",
+            },
+            "& .Mui-icon": {
+              fontSize: "3px",
+            },
+            "& .MuiOutlinedInput-notchedOutline": {
+              borderColor: "transparent",
+              "& legend": {
+                width: 0,
+              },
+            },
+            "& .MuiInputBase-input": {
+              fontFamily: "Rubik",
+              fontSize: 16,
+              fontWeight: 400,
+              color: "#515151",
+              borderColor: "transparent",
+              borderRadius: "3px",
+              padding: "9px 14px",
+              "&::placeholder": {
+                opacity: 0.6,
+              },
+            },
+            "& .MuiSelect-icon": {
+              color: "#515151",
+              fontSize: "large",
+            },
+          }}
+          inputProps={{
+            "data-testid": "population-criterion-input",
```

Comment:
why are we adding all the properties of select again. aren't they already present in the design systems ?

Reply:
this is a slightly different select box with no borders. It is kind of unsetting the default CSS.